src/components/Root/Root.tsx

```diff
@@ -39,21 +44,10 @@ const RootLoader = ({ setAppIsReady }: { setAppIsReady: React.Dispatch<React.Set
   const IS_DEMO_OR_PREVIEW = IS_DEMO_MODE || IS_PREVIEW_MODE;
 
   // Show the spinner while loading except in demo mode (the demo config shows its own loading status)
-  if (settingsQuery.isLoading || (!IS_DEMO_OR_PREVIEW && configQuery.isLoading)) {
+  if (!IS_DEMO_OR_PREVIEW && configQuery.isLoading) {
```

Comment:
Don't you need to re-add the settingsQuery stuff here and below?

Reply:
Yeah, I do need. Working on settings now (and features).